lambdas/src/main/kotlin/scorcerer/server/events/UserCreationEventHandler.kt

```diff
@@ -36,6 +45,31 @@ class UserCreationEventHandler : RequestHandler<SQSEvent, Unit> {
                     row[this.livePoints] = 0
                 }
             }
+
+            try {
+                transaction {
+                    LeagueTable.insert {
+                        it[this.name] = "Global"
+                        it[this.id] = "global"
+                    }
+                }
+            } catch (e: PSQLException) {
+                log.info("Global league exists: $e")
+            }
+
+            transaction {
+                LeagueMembershipTable.insert {
+                    it[this.memberId] = userCreationEvent.id
+                    it[this.leagueId] = leagueId
+                }
+            }
+
+            val globalLeaderboard = caclulateGlobalLeaderboard()
+            val s3Client = S3Client { region = "eu-west-2" }
+            runBlocking {
+                // TODO: add logic to calculate match day
+                LeaderboardS3Service(s3Client, Environment.LeaderboardBucketName).writeLeaderboard(globalLeaderboard, 1)
+            }
```

Comment:
I think the main issue with this is race conditions. If we're processing two user creation events in two separate lambdas then we run the risk of overriding one of the results